src/renderers/shared/stack/reconciler/ReactCompositeComponent.js

```diff
@@ -464,7 +464,8 @@ var ReactCompositeComponent = {
     hostParent,
     hostContainerInfo,
     transaction,
-    context
+    context,
+    shallowRendering
```

Comment:
I don't think this approach is the right way to implement this. The reconciler shouldn't be aware of the current renderer. If this were to be added to React it should be contained within `ReactShallowRenderer`.